Assets/_Application/Layers/LayerTypes/Credentials/Properties/CredentialsValidationPropertySection.cs

```diff
@@ -1,39 +1,47 @@
+using Netherlands3D.Credentials;
 using UnityEngine;
 
 namespace Netherlands3D.Twin.Layers.LayerTypes.Credentials.Properties
 {
-    public class CredentialsValidationPropertySection : MonoBehaviour, ILayerCredentialInterface
+    public class CredentialsValidationPropertySection : MonoBehaviour, ICredentialsPropertySection
     {
-        private LayerCredentialsHandler handler;
+        private ICredentialHandler handler;
 
         [SerializeField] private GameObject validCredentialsPanel;
         [SerializeField] private GameObject invalidCredentialsPanel;
 
-        public LayerCredentialsHandler Handler
+        public ICredentialHandler Handler
         {
             get => handler;
             set
             {
-                if (handler)
+                if(handler != null)
                     handler.CredentialsAccepted.RemoveListener(OnCredentialsAccepted);
 
                 handler = value;
+                if (!handler.StatusEnabled) return;
 
                 OnCredentialsAccepted(handler.HasValidCredentials);
-
                 handler.CredentialsAccepted.AddListener(OnCredentialsAccepted);
             }
         }
 
         private void Start()
         {
-            if (handler)
+            //validation/status panel should be disabled so no handler would be present (maybe validation panel should not be included in the credential input prefab?)            
+            if (handler == null || !handler.StatusEnabled)
+            {
+                gameObject.SetActive(false);
+                return;
+            }
+
+            if (handler != null)
                 handler.CredentialsAccepted.AddListener(OnCredentialsAccepted);
         }
 
         private void OnDestroy()
         {
-            if (handler)
+            if (handler != null)
```

Comment:
De `handler?.SetAuthorizationInputType((AuthorizationType)type);` shortcut maakt het iets minder breed met die if-statements. Dan is de if namelijk niet meer nodig. Tevens: kunnen we braces gebruiken bij alle if-statement die niet een control flow break uitvoeren? (dus: break, continue, return). Ik heb iets teveel meegemaakt dat ontwikkelaars eroverheen lazen dat dit deel van een if is, of denken iets toe te voegen aan de if maar niet door hebben dat er geen braces zijn